### ISSUE TITLE
Encoding large (Type 5) messages causes NMEA max message length to exceed 82 characters.

### ISSUE THREAD

**Issue by bgercken** [M0r13n/pyais#81]
Was running into an issue in encode.py ais_to_nmea_0183 where it was causing messages to exceed the NMEA max size.

On line 54 (in encode.py) I changed max_len to 60, which seems to have resolved it.

Thanks.

**Comment by M0r13n** [comment]
@bgercken Thank you for bringing this up. I will look at this but as soon as time allows. But it might take a couple of days.

**Comment by bgercken** [comment]
No hurry on my part. Thank you very much.

**Comment by M0r13n** [comment]
@bgercken 

Could you provide me some example that led to this issue? I am not able to reproduce the described behavior.

**Comment by bgercken** [comment]
@M0r13n  - Leon, my apologies. It appears that I must have an error in my code. I wrote a test for my data that passed and then added some additional debug to my code and the length of the sentence that you create while encoding is correct. Sorry for taking up some of your time. Regards.

```
data: {'type': 5, 'repeat': 0, 'mmsi': '538008122', 'ais_version': 1, 'imo': 9500091, 'callsign': 'V7GS3', 'shipname': 'CLIPPER BAROLO', 'shiptype': 70, 'to_bow': 166, 'to_stern': 30, 'to_port': 22, 'to_starboard': 10, 'epfd': 1, 'month': 5, 'day': 16, 'hour': 0, 'minute': 30, 'draught': 6.6, 'destination': 'BALTIMORE', 'dte': 0}

sentence: !AIVDO,2,1,0,A,5815J>T2@uKeKLM?<00<hU10E:0858tht0000000DhNF:5H0N@PPC52CClQ@0,0*28 (81)

sentence: !AIVDO,2,2,0,A,0000000000,2*16 (30)
```

**Comment by bgercken** [comment]
I had closed this but I was wrong. Not sure that I can reopen...

Updated test fails:

```
from pyais import encode_dict

data = {
    'type': 5, 'repeat': 0, 'mmsi': '259725000', 'ais_version': 1, 'imo': 9103128, 'callsign': 'LAXP4',
    'shipname': 'STAR HANSA', 'shiptype': 70, 'to_bow': 177, 'to_stern': 22, 'to_port': 17, 'to_starboard': 14,
    'epfd': 1, 'month': 5, 'day': 20, 'hour': 6, 'minute': 0, 'draught': 7.4, 'destination': 'US BAL', 'dte': 0
}


# Maximum sentence length, including the $ and <CR><LF> is 82 bytes.
NMEA_MAX_LEN = 82
NMEA_TERMINATOR = "\r\n"


def test_encode_type5():
    encoded = encode_dict(data)
    for i in range(len(encoded)):
        encoded[i] += NMEA_TERMINATOR
        assert len(encoded[i]) < NMEA_MAX_LEN


if __name__ == "__main__":
    test_encode_type5()
```

**Comment by M0r13n** [comment]
Thank you. I am pretty sure that I forgot to include the final <LF> terminator when calculating the total length of a message. I will fix this tomorrow.

**Comment by M0r13n** [comment]
@bgercken 

You were right. The max payload length should be set to 60 and not 61. The docstring of the relevant method actually mentions [this](https://github.com/M0r13n/pyais/blob/9a49a33890f30a2827ff4dafdfa7a6d924cab539/pyais/encode.py#L41):

```txt
This method takes care of splitting large payloads (larger than 60 characters)
into multiple sentences. With a total of 80 maximum chars excluding end of line
per sentence, and 20 chars head + tail in the nmea 0183 carrier protocol, 60
chars remain for the actual payload.
```

But then the implementation sets `max_len` to 61 ([see](https://github.com/M0r13n/pyais/blob/9a49a33890f30a2827ff4dafdfa7a6d924cab539/pyais/encode.py#L54)).

**Comment by M0r13n** [comment]
Should be fixed with version 2.2.1
